docs/userGuide/siteConfiguration.md

```diff
@@ -37,12 +37,12 @@ Let's examine a typical `site.json` file:
-| **faviconPath** | The location of the favicon. Default: `favicon.ico`. Note: If the favicon was recently changed, when viewing the website, you may need to force-refresh to see the new favicon. |
-| **titlePrefix** | The prefix for all page titles. This prefix will be prepended to all rendered pages. |
-| **pages** | An array of the pages config tells MarkBind all the source files that needs to be rendered. The `src` is the page to be rendered; `title` is the page title for the generated web page. Titles specified in this array will take priority over titles specified within the front matter of individual pages. In addition, globs can be used to define pages that will be rendered. Front matter for the pages will be automatically indexed. However, if front matter or `title` is not specified, the page will have a default title that consists only of the `titlePrefix`. |
-| **ignore** | Files to be ignored when building the website. By default, MarkBind will copy all the assets into the output folder. The ignore pattern follows the pattern used in [`.gitignore`](https://git-scm.com/docs/gitignore#_pattern_format). You may want to ignore all markdown source files by adding the entry `*.md`, as well as the Git working directory `.git/*`. |
-| **deploy** | Settings for the auto Github page deployment. Please refer this [doc](ghpagesDeployment.html) for more details. |
+| **baseUrl** | The base url relative to your domain. Default: <code></code>&nbsp;(empty). For example, if you are using Github Pages to host your deployed website and it is published at `https://markbind.github.io/site-demo-cs2103`, then your `baseUrl` would be `/site-demo-cs2103`. You can use this variable for [specifying path reference](contentAuthoring.html#specifying-path-reference) of images and links. **You may need to change the `baseUrl` when deploying to a different repo.** |
+| **faviconPath** | The location of the favicon. Default: `favicon.ico`. If the favicon was recently changed, you may need to force-refresh to see the new image. |
+| **titlePrefix** | The prefix for all page titles. The separator <code>-</code> will be inserted by MarkBind. |
+| **pages** | An array of pages to be rendered. The `src` is the file; `title` is the page title for the generated web page. Titles specified here take priority over titles specified in the [front matter](contentAuthoring.html#front-matter) of individual pages. Alternatively, `glob` can be used to define a file pattern. If `title` is not specified in the front matter, the page will have `titlePrefix` as its title. |

```

Comment:
I feel that this could use a bit of rearrangement (from "Titles specified here" onwards: it talks about titles, then jumps to `glob`, then suddenly jumps back to title).

Something like this:
```
An array of pages to be rendered. The `src` is the file; `title` is the page title for the generated web page. Alternatively, `glob` can be used to define a file pattern. Titles specified here take priority over titles specified in the [front matter](contentAuthoring.html#front-matter) of individual pages. If `title` is not specified in the front matter, the page will have `titlePrefix` as its title. 
```

Reply:
I considered that, but `title` is not used with `glob`. And the last line may imply: "If `title` is not specified in the front matter (even if it's in **site.json**), the page will have `titlePrefix` as its title."